README.md

```diff
@@ -111,6 +111,26 @@ If you'd like to upgrade to a specific version, run the following:
 
 See [CONTRIBUTING.md](CONTRIBUTING.md)
 
+## Releases
+
+After each minor Crossplane release, a corresponding patched and hardened
+version of Universal Crossplane will be released after exactly 2 weeks.
```

Comment:
How would you feel about softening this wording a bit to accommodate for releasing sooner than 2 weeks in some cases?

Reply:
Sounds good, just added.